taker-frontend/src/App.tsx

```diff
@@ -68,10 +79,47 @@ export const BG_LIGHT = "gray.50";
 export const BG_DARK = "gray.800";
 export const FAQ_URL = "http://faq.itchysats.network";
 
+const fetchGithubVersion = async (setGithubVersion: (version: SemVer | null) => void) => {
+    let response = await fetch("https://api.github.com/repos/itchysats/itchysats/releases/latest", {
+        headers: { accept: "application/json" },
+    });
+
+    let githubVersion = await response.json() as GithubVersion;
+    if (githubVersion?.tag_name) {
+        let version = semver.parse(githubVersion?.tag_name);
+        if (!version) {
+            // needed because 0.20.1 is not a valid semver version
```

Comment:
If you look at the screenshot, it was able to parse the tag correctly as `0.4.20-217-g4d...`

The screenshot was a hack to get it up, i.e. I turned the condition temporarily around 😬 Otherwise it would not show it.

Reply:
> If you look at the screenshot, it was able to parse the tag correctly as `0.4.20-217-g4d...`
> 
> The screenshot was a hack to get it up, i.e. I turned the condition temporarily around 😬 Otherwise it would not show it.

really? that's good! in the rust `semver` crate the comparison would certainly show it (it treats the tag as prerelease, which should happen before the official release).